doc/releases/tentacle.rst

```diff
@@ -0,0 +1,395 @@
+========
+Tentacle
+========
+
+Tentacle is the 20th stable release of Ceph.
```

Comment:
One might quibble re whether the prior cadence's releases like Kraken count as stable, but it's probably best that we align the ordinal number here with the cardinal.